### DISCUSSION TITLE
Parcel tries to resolve packages from wrong folder

### DISCUSSION THREAD

**Discussion by StijnCoolen** [parcel-bundler/parcel#5748] [Q&A]
When running parcel watch the following happens:

```
$ parcel watch src/js/app.js --dist-dir web/dist
🚨 Build failed.
@parcel/transformer-postcss: Could not resolve module "postccs-nested" from "/Users/stijncoolen/Documents/Development/projectname/src/css"
Autoinstall is disabled, please install this package manually and restart Parcel.
```

I've also seen this happen with other modules (postcss-import). Is there anything I can do to point parcel to the right folder (node_modules)?

**Comment by mischnic** [comment]
Can you share a reproduction? Seems to work fine for me.

**Reply by StijnCoolen** [reply]
Not sure if I can easily reproduce the project's settings but it's based on the following:

**package.json**
```
{
  "dependencies": {
    "autoprefixer": "^10.2.4",
    "parcel": "^2.0.0-nightly.562",
    "postcss-import": "^14.0.0",
    "postcss-nested": "^5.0.3"
  },
  "scripts": {
    "dev": "parcel watch src/js/app.js --dist-dir web/dist --public-url ./",
    "build": "parcel build src/js/app.js --dist-dir web/dist --public-url ./"
  }
}
```

**.postcssrc**
```
{
  "plugins": {
    "postccs-import": {},
    "postccs-nested": {},
    "autoprefixer": {
      "grid": true
    }
  }
}
```

**app.js**
```
import '../css/app.css'

console.log('This is a logged message!')
```

**app.css**
```
@import 'text.css'

body {
    background: linear-gradient(to bottom, white, black);
    h1 {
        color: pink;
    }
}
```

**Reply by mischnic** [reply]
You have a typo here:
```json
    "postccs-import": {},
    "postccs-nested": {},
```

postccs instead of postcss.

There should definitely be a better error message: https://github.com/parcel-bundler/parcel/issues/5750

**Reply by StijnCoolen** [reply]
Oops, my fault... this does fix the importing error but now the exported css seems to be empty.

**Reply by mischnic** [reply]
Even after clearing the cache? The error message from before might have screwed it up
